WebRoot/js/ajax/dwt/widgets/DwtControl.js

```diff
@@ -1139,13 +1143,22 @@ function(state) {
     this._displayState = state;
     Dwt.delClass(this.getHtmlElement(), DwtControl._RE_STATES, state);
 
-    AjxUtil.foreach(DwtControl._ARIA_STATES, (function(attribute, state) {
-        if (DwtControl._RE_STATE[state].test(this._displayState)) {
-            this.setAttribute(attribute, true);
-        } else {
-            this.removeAttribute(attribute);
-        }
-    }).bind(this));
+	// Set Aria states.
+	var ariaStateKeys = Object.keys(DwtControl._ARIA_STATES);
+	var attribute = DwtControl._ARIA_STATES[state];
+
+	// Add/remove aria states only if given state is blank or given state is present in _ARIA_STATES
+	// state will be blank when user navigate to other app. e.g. User is on Mail and navigate to Calendar then state='' for Mail app object.
+	if (attribute || state === '') {
+		for (var i=0; i < ariaStateKeys.length; i++) {
```

Comment:
`var i = 0;`
Please check the indentation. The code that was replaced had a different indentation.